src/main/kotlin/classes/IssueClass.kt

```diff
@@ -0,0 +1,28 @@
+data class IssueClass(
```

Comment:
use more consistent naming for the rest, they should all either have the Class suffix or none at all